Astro release notes - August 17 (#1044)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.3.0 ([Release notes](cli/release-notes.md))

## August 18, 2022

> ### Save on Deployment resource usage with worker queues

Reviewer: ```suggestion
### Optimize your Deployment for different tasks with worker queues
```

**Latest CLI Version**: 1.3.0 ([Release notes](cli/release-notes.md))

## August 18, 2022

### Worker queues

Worker queues are a new way to size your workers on Astro and configure your Deployment to best fit the needs of your tasks. A worker queue is a set of configurations that apply to a group of workers in your Deployment. Within a worker queue, you can configure worker type and size as well as autoscaling behavior.

> In the Cloud UI, you can now size workers more simply and create multiple worker queues.

Author: ```suggestion
In the Cloud UI, you can now scale workers more efficiently by creating multiple worker queues.
```
